lib/testutils/rules.go

```diff
@@ -20,82 +20,81 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-var ipv4 = 4
-var ipv6 = 6
-var strProtocol1 = numorstring.ProtocolFromString("icmp")
-var strProtocol2 = numorstring.ProtocolFromString("udp")
-var numProtocol1 = numorstring.ProtocolFromInt(240)
-
-var portRange, singlePort, namedPort numorstring.Port
+var InRule1, InRule2, EgressRule1, EgressRule2 apiv2.Rule
 
 func init() {
-	var err error
-	portRange, err = numorstring.PortFromRange(10, 20)
+	portRange, err := numorstring.PortFromRange(10, 20)
 	if err != nil {
 		logrus.WithError(err).Panic("Failed to create port range")
 	}
-	singlePort = numorstring.SinglePort(1024)
-	namedPort = numorstring.NamedPort("named-port")
-}
+	singlePort := numorstring.SinglePort(1024)
+	namedPort := numorstring.NamedPort("named-port")
 
-var icmpType1 = 100
-var icmpCode1 = 200
+	ipv4 := 4
+	ipv6 := 6
+	strProtocol1 := numorstring.ProtocolFromString("tcp")
+	strProtocol2 := numorstring.ProtocolFromString("udp")
+	numProtocol1 := numorstring.ProtocolFromInt(240)
 
-var cidr1 = "10.0.0.0/24"
-var cidr2 = "20.0.0.0/24"
-var cidrv61 = "abcd:5555::/120"
-var cidrv62 = "abcd:2345::/120"
+	icmpType1 := 100
+	icmpCode1 := 200
 
-var icmp1 = apiv2.ICMPFields{
-	Type: &icmpType1,
-	Code: &icmpCode1,
-}
+	cidr1 := "10.0.0.0/24"
+	cidr2 := "20.0.0.0/24"
+	cidrv61 := "abcd:5555::/120"
+	cidrv62 := "abcd:2345::/120"
+
+	icmp1 := apiv2.ICMPFields{
+		Type: &icmpType1,
+		Code: &icmpCode1,
+	}
 
-var InRule1 = apiv2.Rule{
-	Action:    "allow",
-	IPVersion: &ipv4,
-	Protocol:  &strProtocol1,
-	ICMP:      &icmp1,
-	Source: apiv2.EntityRule{
-		Nets:     []string{cidr1},
-		Selector: "label1 == 'value1'",
-		Ports: []numorstring.Port{
-			portRange,
-			singlePort,
-			namedPort,
+	InRule1 = apiv2.Rule{
```

Comment:
the reason i had to pull these inside the init() was because when it's a var outside an init() golang initializes the vars with the zero values for some reason, so it may look unrelated but the tests wont work without these things being inside the init() function